cmd/cephcsi.go

```diff
@@ -63,6 +63,7 @@ func init() {
 
 	// cephfs related flags
 	flag.StringVar(&conf.MountCacheDir, "mountcachedir", "", "mount info cache save dir")
+	flag.BoolVar(&conf.ForceKernelClient, "forcecephkernelclient", false, "enable Ceph Kernel clients on kernel < 4.17 which support quotas")
```

Comment:
which does not support quotas?

Reply:
No, the flag is to enable the kernel client on kernels with version <4.17 that do support quotas (via backported patches). All kernels >=4.17 support quotas and don't need the flag